includes/enrolment/class-sensei-course-enrolment.php

```diff
@@ -14,6 +14,7 @@
  */
```

Comment:
I took a stab at this here: e78b7ec2443e236b72556f73c246459e959596ea

In doing so, I made a small change here to standardize one type of invalidation by setting the user meta to an empty string versus removing. I think setting to an empty string makes sense because it can be differentiated from a result that just hasn't been calculated versus one that has been intentionally invalidated. 

I also realized something else... I think for the version hash, we should use the version of _all_ providers instead of just the ones that have said they handle this course. The reason being, a provider might change their logic for whether or not they handle a course. I'll put this up in a new PR really quick.

Reply:
Nice!